src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -223,6 +230,134 @@ void refuseAttemptToMoveRequestBeingFulfilledFromOneOfTheTopPositionsWhenTlrEnab
       is("Requests can not be displaced from top positions when fulfillment begun."));
   }
 
+  @Test
+  void shouldGetRequestQueueForItemSuccessfully() {
+    UUID facultyGroupId = patronGroupsFixture.faculty().getId();
+    UUID staffGroupId = patronGroupsFixture.staff().getId();
+    UUID isbnIdentifierId = identifierTypesFixture.isbn().getId();
+
+    final ItemResource smallAngryPlanet = itemsFixture
+      .basedUponSmallAngryPlanet(
+        identity(),
+        instanceBuilder -> instanceBuilder.addIdentifier(isbnIdentifierId, "9780866989732").withId(instanceId),
+        itemBuilder -> itemBuilder
+          .withCallNumber("itCn", "itCnPrefix", "itCnSuffix")
+          .withEnumeration("enumeration1")
+          .withChronology("chronology")
+          .withVolume("vol.1")
+          .withCopyNumber("1"),
+        "649113164644");
+
+    final IndividualResource sponsor = usersFixture.rebecca(user -> user.withPatronGroupId(facultyGroupId));
+    final IndividualResource proxy = usersFixture.undergradHenry(user -> user.withPatronGroupId(staffGroupId));
+    proxyRelationshipsFixture.nonExpiringProxyFor(sponsor, proxy);
```

Comment:
Do we really need these variables?

Reply:
yes, I think, because these are added new properties on the extendedRepresentation()